File: c#/types/strings/SubstringModification.cs
class SubstringModification {
  public static void Main(string[] args) {
    string cite = "Stat rosa pristina nomine, nomina nuda tenemus.";
    System.Console.WriteLine(string.Format("cite = {0}\n", cite));

    string modified_cite = cite.Replace("rosa", "odonata");

    System.Console.WriteLine(string.Format("modified cite = {0}\n", modified_cite));
  }
}

File: java/types/strings/SubstringModification.java
class SubstringModification {
  public static void main(String[] args) {
    String cite = "Stat rosa pristina nomine, nomina nuda tenemus.";
    System.out.println(String.format("cite = %s\n", cite));

    String modified_cite = cite.replace("rosa", "odonata");

    System.out.println(String.format("modified_cite = %s\n", modified_cite));
  }
}
